phpunit.xml.dist

```diff
@@ -3,7 +3,6 @@
     colors="true"
     processIsolation="false"
```

Comment:
All those false ones can go away, as they are redundant.

Reply:
Have removed the two false options mentioned.